src/validations.ts

```diff
@@ -1,13 +1,39 @@
-import { LogLevel } from './logs';
-import { IPathPosition } from './parsers';
+import { IPathPosition, ObjPath } from './parsers';
+
+export enum ValidationSeverity {
+  Info = 'info',
+  Warn = 'warn',
+  Error = 'error',
+}
+
+export enum ValidationSeverityNum {
+  Info = 30,
+  Warn = 40,
+  Error = 50,
+}
 
 export interface IValidation {
```

Comment:
This validation is for the graph, correct? Not prism?

Reply:
Everywhere.

Reply:
Sounds good, this pull request looks good to me!